src/test/lambda/wizards/uploadLambdaWizard.test.ts

```diff
@@ -65,4 +67,27 @@ describe('UploadLambdaWizard', function () {
             tester.confirmedDeploy.assertShow(3)
         })
     })
+
+    describe('invoked from template', function () {
+        let tempDir: string
+        let tempDirUri: vscode.Uri
+        let invokePath: vscode.Uri
+        beforeEach(async function () {
+            tempDir = await makeTemporaryToolkitFolder()
```

Comment:
I think we should probably cleanup this folder once we are done with it. It also looks like a couple other tests in this file aren't cleaning up after themselves either 🤔

Reply:
Cleaned up test folders